src/dao/ContractsManagerDAO.js

```diff
@@ -20,6 +21,7 @@ const DAO_VOTE = 6
 const DAO_REWARDS = 7
 const DAO_ASSETS_MANAGER = 8
 const DAO_TIME_HOLDER = 9
+const DAO_ASSET_DONATOR = 10
```

Comment:
this contract is not in the contract ContractsManagerInterface enum ContractType, 10 is busy by CrowdsaleManager, (see https://github.com/ChronoBank/SmartContracts/blob/develop/contracts/ContractsManagerInterface.sol)

use here string value such as 'erc20' for DAO_ERC20